pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/lifecycle_policy/guide.en-gb.md

```diff
@@ -119,18 +119,17 @@ VMware extended the support of NSX Datacenter for vSphere on OVHcloud infrastruc
 
 ### Veeam Backup & Replication lifecycle policy
 
-To know the lifecycle policy of Veeam products, please refer to the editor's publications accessible at the following URLs:
+To know the lifecycle policy of Veeam products, please refer to the editor's publications accessible at the following URLs :
 
-[Veeam Version release](https://www.veeam.com/releasestatus_rn.pdf)
-
-[Veeam Product Version Lifecycle](https://www.veeam.com/product-lifecycle.html)
+- [Veeam Version release](https://www.veeam.com/releasestatus_rn.pdf)
+- [Veeam Product Version Lifecycle](https://www.veeam.com/product-lifecycle.html)
 
 ### Zerto lifecycle policy
 
-To know the lifecycle policy of Zerto products, please refer to the editor's publication accessible at the following URL:
+To know the lifecycle policy of Zerto products, please refer to the editor's publication accessible at the following URL :
 
-[Zerto Virtual Replication Product Version Lifecycle Matrix](https://help.zerto.com/bundle/Lifecycle.Matrix.HTML/page/Content/Lifecycle_Matrix/Lifecycle_Matrix.htm#zerto_virtual_replication_product_version_lifecycle_matrix_r_893035264_1010900)
+- [Zerto Virtual Replication Product Version Lifecycle Matrix](https://help.zerto.com/bundle/Lifecycle.Matrix.HTML/page/Content/Lifecycle_Matrix/Lifecycle_Matrix.htm#zerto_virtual_replication_product_version_lifecycle_matrix_r_893035264_1010900)
 
 ## Go further
 
-Join our community of users on <https://community.ovh.com/en/>.
+Join our [community of users](/links/manager).
```

Comment:
wrong link on all versions except fr-fr and fr-ca
It should be `(/links/community)`